features/lege-waarden.feature

```diff
@@ -1,10 +1,27 @@
 # language: nl
 
+@post-assert
 Functionaliteit: Leveren van lege waarden
 
   Rule: Een boolean veld met waarde false wordt niet geleverd
 
-    Abstract Scenario: persoon heeft veld: 'indicatie geheim (70.10)'
+    @gba
+    Scenario: persoon heeft veld: 'indicatie geheim (70.10)'
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                     | waarde    |
+      | burgerservicenummer      | 555550001 |
+      | indicatie geheim (70.10) | 0         |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                            |
+      | type                | RaadpleegMetBurgerservicenummer                   |
+      | burgerservicenummer | 555550001                                         |
+      | fields              | burgerservicenummer,geheimhoudingPersoonsgegevens |
+      Dan heeft de response een persoon met alleen de volgende gegevens
+      | naam                          | waarde    |
+      | burgerservicenummer           | 555550001 |
+      | geheimhoudingPersoonsgegevens | 0         |
```

Comment:
De property 'geheimhoudingPersoonsgegevens' heeft hier de waarde '0' wat volgens mij betekent dat deze 'false' is. Klopt het dan wel dat deze property hier geleverd wordt?